system_metrics_collector/src/system_metrics_collector/linux_process_cpu_measurement_node.cpp

```diff
@@ -75,3 +82,11 @@ std::string LinuxProcessCpuMeasurementNode::GetMetricName() const
 }
 
 }   // namespace system_metrics_collector
+
+
+#include "rclcpp_components/register_node_macro.hpp"
```

Comment:
Include at the top of the file? Why here?

Reply:
I have moved it to the top of the file.